hermes-2.cxx

```diff
@@ -1080,7 +1238,7 @@ int Hermes::rhs(BoutReal t) {
           } else {
```

Comment:
Should the line above (l.1237, but Github won't let me comment on it because it's not close enough to a changed line) be `phi_boundary3d`, or have a call to `phi.setBoundaryTo(phi_boundary3d)` before it?